README.md

```diff
@@ -28,9 +28,8 @@ The V2X Hub system reduces time needed to create and deploy a roadside based V2X
 * ODE Forward Plugin – Pushes data to an operational data environment server using a known IP address and port.
 * Pedestrian Plugin – Creates a PSM using information obtained from nomadic devices (ex. cell phones) through a local webserver.
 * Preemption Plugin – Calls a preemption table on a controller using NTCIP 1202 V3 commands to provide passage to an emergency vehicle upon request through BSMs
-* SPAT Logger Plugin – Logs V2X Hub generated, UPER-encoded SPAT messages in a .csv file along with a system timestamp
 * TIM Plugin – Creates and broadcasts a TIM message from an .xml file based on user input through GUI or local webserver
-* BSM Logger Plugin – Decodes and logs BSMs received by the Message Receiver Plugin to a .csv file
+* Message Logger Plugin – Decodes and logs BSMs and SPaTs received by the Message Receiver Plugin to a .json file
```

Comment:
Spat messages are not received by the message receiver plugin. Maybe remove that part of the sentence. Also doesn't it create a json and a hex encoded file.?

Reply:
The hex is in the json, but there's also a bin file that's generated. Made the changes and pushed